shims/spark33/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormatWriter.scala

```diff
@@ -100,6 +100,32 @@ object FileFormatWriter extends Logging {
    * @return
    *   The set of all partition paths that were updated during this write job.
    */
+
+  // scalastyle:off argcount
+  def write(
```

Comment:
@lwz9103 Can we also support in spark 32?

Reply:
It's a good suggestion, and it has been added. Thanks.